editor/src/clj/editor/util.clj

```diff
@@ -133,14 +133,14 @@
        (.compare c (f a) (f b))))))
 
 (defn os-raw
-  "Returns :win32, :darwin or :linux"
+  "Returns :win32, :macos or :linux"
```

Comment:
I think this type of editor code shouldn't need a duplicate of the PLatform.java.
But that's for the future tasks...